simplq/src/components/pages/Home/LandingPage.jsx

```diff
@@ -1,16 +1,29 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
 import Header from 'components/common/Header';
 import QueueForm from 'components/common/CreateJoinForm';
+import { useGetUserQueues } from 'store/asyncActions';
+import { useDispatch } from 'react-redux';
 import styles from './Home.module.scss';
 import MyQueues from './MyQueues';
 
 export default () => {
   let subtitle = 'A long overdue alternative to physical queues';
-  const { user, isAuthenticated } = useAuth0();
+  const { user, isAuthenticated, isLoading } = useAuth0();
   if (isAuthenticated) {
     subtitle = `Hi ${user.name}, welcome back!`;
   }
+  const dispatch = useDispatch();
+  const getUserQueues = useGetUserQueues();
+
+  useEffect(() => {
+    // TODO: Server still returns queues for anonymous users
+    //  Dispatch action only if auth is loaded
```

Comment:
I wanted to render the app content in Layout only after auth0's isLoading is done, so that these kind of issues do not happen.

That's why I had wrapped the Routes entirely in a Loading component here:

![image](https://user-images.githubusercontent.com/11256376/109376479-b1775400-78ea-11eb-92b8-3982a430fd84.png)

I will take a look once this is merged. 🤔

Reply:
I'll see if this approach works and add comments in the Layout component if it did..

Reply:
I didn't completely understand what you meant, but yeah, if you think something's lacking, we can figure it out in a PR.